iot-hub/Samples/device/AndroidSample/things/build.gradle

```diff
@@ -0,0 +1,58 @@
+apply plugin: 'com.android.application'
+
+//***********************************************************************************************//
+def IOTHUB_DEVICE_CONNECTION_STRING_ENV_VAR_NAME = project.hasProperty('DeviceConnectionString') ? '"'+project.property('DeviceConnectionString')+'"' : '"Define Device Connection"'
+def STRING='String'
+//***********************************************************************************************//
+
+android {
+    compileSdkVersion 27
+
+
+    defaultConfig {
+        applicationId "android.iot.sdk.azure.microsoft.com.things"
+        minSdkVersion 24
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        each {
+            //buildTypes.mBuildConfigFields 'DATATYPE','VARIABLE',|"GRADLE VARIABLE|"'
+            buildConfigField STRING, 'DeviceConnectionString', IOTHUB_DEVICE_CONNECTION_STRING_ENV_VAR_NAME
+        }
```

Comment:
Can you keep track of this concern with app studio ? We should call this out to them.

Reply:
what are you referring to?